modin/pandas/test/test_series.py

```diff
@@ -770,7 +770,14 @@ def test_align(data):
 )
 def test_all(data, skipna):
     modin_series, pandas_series = create_test_series(data)
-    df_equals(modin_series.all(skipna=skipna), pandas_series.all(skipna=skipna))
+    try:
+        pandas_result = pandas_series.all(skipna=skipna)
+    except Exception as e:
+        with pytest.raises(type(e)):
+            modin_series.all(skipna=skipna)
+    else:
+        modin_result = modin_series.all(skipna=skipna)
+        df_equals(modin_result, pandas_result)
```

Comment:
```suggestion
    eval_general(*create_test_series(data), lambda df: df.all(skipna=skipna))
```
`eval_general` can check the equality of thrown errors from modin and pandas.

Also, remove `modin_series, pandas_series = create_test_series(data)` above (I can't comment unchanged code)

Reply:
Fixed.